synapse/groups/groups_server.py

```diff
@@ -404,9 +404,16 @@ def get_group_profile(self, group_id, requester_user_id):
 
         yield self.check_group_is_ours(group_id, requester_user_id)
 
-        group_description = yield self.store.get_group(group_id)
+        group = yield self.store.get_group(group_id)
+
+        if group:
+            cols = [
+                "name", "short_description", "long_description",
+                "avatar_url", "is_public",
+            ]
+            group_description = { key: group[key] for key in cols }
+            group_description["is_openly_joinable"] = join_policy == "open"
```

Comment:
@lukebarnard1 you missed `join_policy = group['join_policy']` or similar

Reply:
@krombel thank you - I noticed my sytests were failing.